.travis.yml

```diff
@@ -1,19 +1,22 @@
 language: php
+
 php:
   - 5.3
   - 5.4
   - 5.5
+  - 5.6
+  - 7.0
   - hhvm
 
-before_script:
-  - composer self-update
-  - composer install --prefer-source --no-interaction --dev
-
-branches:
-  only:
-    - master
-
 matrix:
   allow_failures:
     - php: hhvm
+    - php: 7.0
   fast_finish: true
+
+before_script:
+  - composer self-update
```

Comment:
No, but it's a waste of resources. There's no specific reason why a slightly older version of composer isn't just fine.